packages/aws-cdk-lib/aws-efs/lib/efs-file-system.ts

```diff
@@ -42,7 +43,22 @@ export enum LifecyclePolicy {
   /**
    * After 90 days of not being accessed.
    */
-  AFTER_90_DAYS = 'AFTER_90_DAYS'
+  AFTER_90_DAYS = 'AFTER_90_DAYS',
+
+  /**
+   * After 180 days of not being accessed.
+   */
+  AFTER_180_DAYS = 'AFTER_180_DAYS',
+
+  /**
+   * After 270 days of not being accessed.
+   */
+  AFTER_270_DAYS = 'AFTER_270_DAYS',
+
+  /**
+   * After 365 days of not being accessed.
+   */
+  AFTER_365_DAYS = 'AFTER_365_DAYS',
```

Comment:
It looks like `TransitionToIA` and `TransitionToArchive` have the same set of enums excelt that `TransitionToArchive` does not allow `AFTER_1_DAY` [source](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-efs-filesystem-lifecyclepolicy.html)

Can you add in a synth-time check that throws an error if `AFTER_1_DAY` is set for `transitionToArchivePolicy`?  Plus a unit test :)

Reply:
It looks to me that `TransitionToIA` and `TransitionToArchive` both allow `AFTER_1_DAY`, the order is a bit off in the docs...
![Screenshot 2024-01-16 143108](https://github.com/aws/aws-cdk/assets/61606375/591f22ae-b38b-4e0a-a3a5-dd8b79b13466)

Reply:
It looks like they share the same set of enums to me. The order is a bit off for `AFTER_1_DAY`...
![Screenshot 2024-01-16 144600](https://github.com/aws/aws-cdk/assets/61606375/5b6fbe7f-6ebe-4caa-abc5-abd7e48ad6e2)

Reply:
I'm also able to create EFS with `transitionToArchive` set to 1 day as well!

![image](https://github.com/aws/aws-cdk/assets/61606375/3aea6f4e-3bd9-494b-89b9-05f767fa6be9)

Reply:
🤦🤦🤦

Reply:
sorry. the order screwed me up and shame on me for expecting the enum list to be in the same chronological order :)